torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3126,6 +3159,16 @@ def _make_tensor_helper(shape, low=None, high=None):
            skips=(SkipInfo('TestOpInfo', 'test_duplicate_method_tests'),),
            supports_out=False,
            sample_inputs_func=sample_inputs_index_fill),
+    OpInfo('index_copy',
+           dtypes=all_types_and_complex_and(torch.bool, torch.float16, torch.bfloat16),
+           test_inplace_grad=False,
+           supports_out=False,
+           skips=(
+               # https://github.com/pytorch/pytorch/issues/49707
+               # Fails on cpu throwing: RuntimeError: "index_select" not implemented for 'Half' / 'BFloat16'
+               SkipInfo('TestCommon', 'test_variant_consistency_jit', dtypes=[torch.float16, torch.bfloat16]),
```

Comment:
This skip can be removed but a new skip is likely needed for test_supported_backward

Reply:
In fact, https://github.com/pytorch/pytorch/pull/53898 corrected this problem, so now we should be able to get rid of this skip altogether.